docs/v3.x/getting-started/deployment.md

```diff
@@ -70,6 +70,18 @@ Manual guides for deployment on various platforms, for One-click and docker plea
 	</InstallLink>
 </div>
 
+<div>
+	<InstallLink link="../deployment/ibm-cloud">
+    <template #icon>
+			<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 32 32" preserveAspectRatio="xMidYMid meet"><path d="M24.5805,13.65A9.7327,9.7327,0,0,0,6.286,17.5387,6.75,6.75,0,0,0,7,31H23a8.7465,8.7465,0,0,0,1.5805-17.35ZM23,29.5H7a5.2373,5.2373,0,0,1-.6993-10.4293,9.7265,9.7265,0,0,0,1.1036,3.7838.75.75,0,1,0,1.3213-.709A8.25,8.25,0,0,1,22.752,13.5133a8.754,8.754,0,0,0-6.315,2.95.75.75,0,1,0,1.1245.9922A7.25,7.25,0,1,1,23,29.5Z" fill="#fff"/></svg>
```

Comment:
Thank you for finding an SVG (I'll review the docs locally to confirm how it looks :wink: )

Reply:
Thank you, is there any way to "compile" the doc locally? This way I could make a first review.

Reply:
Yeah if you clone the mono-repo locally and cd into the docs and run something like `yarn && yarn dev` or `npm i && npm run dev` (using linux commands here, `&&` won't work on windows) it will spin up the docs on localhost:8080/documentation

Reply:
I've detected and fixed some minor issues during the local check, thank you! :)